### DISCUSSION TITLE
Who are you?

### DISCUSSION THREAD

**Discussion by VentuzBjoernGdanietz** [VentuzTechnology/Ventuz.Extension.prerelease#5] [General]
Please introduce yourself.

**Comment by VentuzBjoernGdanietz** [comment]
I am Bjoern Gdanietz, Senior Developer at Ventuz.

**Comment by VentuzTechnology** [comment]
Daniel Willer, Head of Product at Ventuz

**Comment by le-rou** [comment]
I am Sebastian M. Rouaiha or rou for short. Developer at Glare Productions GmbH.
